esp/src/eclwatch/FileBloomsWidget.js

```diff
@@ -0,0 +1,120 @@
+define([
+    "dojo/_base/declare",
+    "dojo/_base/lang",
+    "dojo/i18n",
+    "dojo/i18n!./nls/hpcc",
+    "dojo/_base/array",
+
+    "dijit/registry",
+
+    "dgrid/selector",
+
+    "hpcc/GridDetailsWidget",
+    "src/WsDfu",
+    "src/ESPUtil",
+    "hpcc/DelayLoadWidget",
+], function (declare, lang, i18n, nlsHPCC, arrayUtil,
+    registry, selector,
+    GridDetailsWidget, WsDfu, ESPUtil, DelayLoadWidget) {
+    return declare("FileBloomsWidget", [GridDetailsWidget, ESPUtil.FormHelper], {
+        i18n: nlsHPCC,
+
+        gridTitle: nlsHPCC.title_Blooms,
+        idProperty: "__hpcc_id",
+
+        init: function (params) {
+            var context = this;
+            if (this.inherited(arguments))
+                return;
+
+            this._refreshActionState();
+            this.refreshGrid();
+        },
+
+        initTab: function () {
+            var currSel = this.getSelectedChild();
+            if (currSel && !currSel.initalized) {
+                if (currSel.id === this.id + "_Grid") {
+                    this.refreshGrid()
+                } else if (currSel.id === this.fileBloomsWidget.id && !this.fileBloomsWidget.initalized) {
+                    this.fileBloomsWidget.init({
+                        firstLoad: true
+                    });
+                } else {
+                    currSel.init(currSel.params);
+                }
+            }
+        },
+
+        postCreate: function (args) {
+            var context = this;
+            this.inherited(arguments);
+            this.openButton = registry.byId(this.id + "Open");
+            this.refreshButton = registry.byId(this.id + "Refresh");
+        },
+
+        createGrid: function (domID) {
+            var context = this;
+            var retVal = new declare([ESPUtil.Grid(true, true)])({
+                store: this.store,
+                columns: {
+                    col1: selector({
+                        width: 10,
+                        selectorType: "checkbox",
+                        disabled: function (item) {
```

Comment:
Probably do not need to disable anything since there is not actions to be taken.